demo/src/main/java/com/midtrans/demo/DemoConfigActivity.java

```diff
@@ -551,6 +552,7 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
             public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
                 if (checked) {
                     cardClickTitle.setText(R.string.credit_card_type_one_click);
+                    secureEnabledSelection.setChecked(true);
```

Comment:
it will automatically change the title when call the method `setOnCheckedChangeListener`